### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by seesleestak**

### CODE REVIEW

**Comment by seesleestak** [comment]
Will do some tests tomorrow

File: packages/hui-pages/src/routes/index.js [hunk 1]
```diff
@@ -27,17 +27,18 @@ import FormsDocs from './FormsDocs'
 import IconDocs from './IconDocs'
```
Comment (billyxs) [inline packages/hui-pages/src/routes/index.js]: Getting an error for line 9 above as there isn't a Sidebar component in hui-pages
`import Sidebar from '../components/Sidebar'`

Comment (seesleestak) [reply packages/hui-pages/src/routes/index.js]: Ah, that would make sense since I renamed it lol

**Comment by billyxs** [comment]
Getting these not so nice looking warnings on tests

sidebar-tests

**Comment by seesleestak** [comment]
Yeah, was going to tackle some actual tests for it today. I'm just not rendering it with the required props in the test suite

**Comment by billyxs** [comment]
Sweet, looks great.

**Comment by seesleestak** [comment]
Added tests. Should be good for another look.

File: packages/sidebar/package.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  "version": "1.0.0",
+  "description": "hixme-ui Sidebar component",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "lib/index.js",
+  "scripts": {
+    "clean": "rimraf lib",
+    "test": "jest",
+    "test:watch": "npm test -- --watch --verbose",
```
Comment (seesleestak) [inline packages/sidebar/package.json]: The `--verbose` flag here helps visualize all the tests that are running a bit better.

Comment (billyxs) [reply packages/sidebar/package.json]: sweet

**Pull request merged**